service/service.go

```diff
@@ -65,6 +65,7 @@ func New(config Config) (*Service, error) {
 	var exporterService *exporter.Service
 	{
 		exporterConfig := exporter.DefaultConfig()
+		exporterConfig.Logger = config.Logger
```

Comment:
Nitpick. The order is usually alphabetical if not logical otherwise, or at least aligned with the structure definition.